metadata.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.25.0 < 9.0.0"
+      "version_requirement": ">= 4.25.0 < 8.0.0"
```

Comment:
This seems unrelated and should be reverted

Reply:
it was just for testing, because without the downgrade i get:

```
  Error: Could not install module 'puppet-nginx' (v3.2.0)
    The requested version cannot satisfy one or more of the following installed modules:
      puppetlabs-stdlib, installed: 8.0.0, expected: >= 5.0.0 < 8.0.0
```

this will be fixed by a new release of puppet-nginx